### ISSUE TITLE
Budgie components/menus/titlebars not scaled

### ISSUE THREAD

**Issue by jwong78** [budgie-desktop/budgie-desktop#1238]
I couldn't see this specific issue reported previously, but I use to be able to set the window scaling factor via the gnome tweak tool. This option has now disappeared and I can now see there is a scaling option via the Displays view under Settings.

However, when I set my monitor to a resolution of 3200x1800 and set the scale to 200%, the budgie desktop components/menus are not scaled. Some applications are scaled (i.e. atom) but the title/menu bars are not.

Is there a setting I am missing or is this something still under development?

Is it expected the tweak tool not to display the window scale option no more?

**Enivironment details below:**
Machine: Dell XPS 13
Operating System: Solus 3.26.1
~->budgie-desktop --version
budgie-desktop 10.4
Copyright © 2014-2017 Budgie Desktop Developers
~->uname -a
Linux devuser 4.9.59-55.lts #1 SMP Fri Oct 27 14:29:55 UTC 2017 x86_64 GNU/Linu

**Comment by MaxJonasG** [comment]
you need to change the font size to scale the header bar (I believe)

**Comment by jwong78** [comment]
> you need to change the font size to scale the header bar (I believe)

Not sure this is the solution. The UI elements do not scale. That only scaled the font.

**Comment by MaxJonasG** [comment]
true. I believe I reported that earlier and there was no solution (yet)
